src/components/gallery/cards/assetCard.vue

```diff
@@ -21,29 +21,30 @@ export default defineComponent({
 <template lang="pug">
 q-card
   q-card-section
+    q-badge.text-subtitle2.float-right(
```

Comment:
General styling. This shouldn't happen. And if it does, it should look better
![image](https://user-images.githubusercontent.com/14140515/176620993-5a52c80a-6641-40ca-a965-76e050008e0a.png)